docs/user/customization.md

```diff
@@ -449,6 +449,72 @@ Example application of `loglevel=7` (change Linux kernel log level to KERN_DEBUG
 
     If you wish to confirm the kernel argument is indeed being applied on the system, you can `oc debug` into a node and check with `rpm-ostree kargs`.
 
+#### Switching RHCOS host kernel using KernelType
+
+With OCP 4.4 and onward release, it is possible to switch from traditional to Real Time (RT) kernel on RHCOS node. During install time, switching to RT kernel can be done through  manifests as an installer operation.  To set kernelType as day 2 operation, see [MachineConfig][machine-config].
+
+Example for switching to RT kernel on worker nodes during initial cluster install:
+
+1. Run `manifests` target to create all the manifests.
```

Comment:
No objection to listing the steps fully, but I would at least link to https://github.com/openshift/installer/blob/ee1b793414178173e03c55b0e6c891849777531c/docs/user/customization.md#install-time-customization-for-machine-configuration

Probably in the first paragraph.  Or we could have a longer blurb about it in the MCO docs here (since it applies to everything, not just the RT kernel).

Reply:
fixed